frontend/packages/knative-plugin/src/utils/__tests__/create-eventsources-utils.spec.ts

```diff
@@ -297,4 +299,83 @@ describe('sanitizeSourceToForm always returns valid Event Source', () => {
     expect(formData.application.name).toEqual('');
     expect(formData.application.selectedKey).toEqual(UNASSIGNED_KEY);
   });
+
```

Comment:
Pls Add the null test as well
```suggestion
  expect(isSecretKeyRefPresent(null)).toBe(false);
```

Reply:
added